chroot-bin/croutonxinitrc-wrapper

```diff
@@ -126,6 +126,15 @@ if [ "$xmethodtype" != "xiwi" ]; then
     fi
 fi
 
+# Apply the Chromebook keyboard map in xiwi if using Freon. 
+if [ "$xmethodtype" = 'xiwi' ] && [ ! -f "/sys/class/tty/tty0/active" ]; then
```

Comment:
This should be an elif for the above block, and the comments above the previous block need to be tweaked.